pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -39,7 +39,7 @@ const (
 	dashboardPortHTTPS             = 8443
 	dashboardPortHTTP              = 7000
 	dashboardUsername              = "admin"
-	dashboardPasswordName          = "rook-ceph-dashboard-password"
+	dashboardPdName                = "rook-ceph-dashboard-password"
```

Comment:
Why?

Reply:
The word Password is trigerring the gosec. I changed the variable name to `dashboardPwdName` and yet it trigerred the gosec as a security issue. A better variable name would be what? @leseb

Reply:
I think it's okay to use `#nosec for this specific line and comment that gosec flags the variable as a potential problem because it has the word `password` in it. We aren't actually using the password, and we know it, so this is safe to ignore. Otherwise devs who come later won't know what `Pd` means.